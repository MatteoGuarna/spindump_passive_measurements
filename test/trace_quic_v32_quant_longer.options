--textual --report-packets --report-spins --report-notes --relative