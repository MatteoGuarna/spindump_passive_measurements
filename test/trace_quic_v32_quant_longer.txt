
November 2020 run with Quant (QUIC version draft-32) client against Quant server, with a slightly longer run than normal, hopefully generating spin events. The run was on Linux. The following command was used:

     ./bin/client -i enp0s3 https://quant.eggert.org/55000
