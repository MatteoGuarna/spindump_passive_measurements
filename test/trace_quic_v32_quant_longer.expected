QUIC 10.30.0.167 <-> 91.190.195.94 049b645a-eb2025df6300a49f (56324:4433) at 0 new starting packets 0 0 bytes 0 0 note "No rsp,v.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 049b645a-eb2025df6300a49f (56324:4433) at 122534 spinvalue starting 0 responder packets 1 0 bytes 1500 0 bandwidth 1500 0 note "No rsp,v.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 049b645a-04b49f0f (56324:4433) at 124552 change starting packets 1 1 bytes 1500 293 bandwidth 1500 293 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 049b645a-04b49f0f (56324:4433) at 124552 measurement up right 124552 packets 1 1 bytes 1500 293 bandwidth 1500 293 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 131965 packet up initiator length 128 packets 2 2 bytes 1628 1793 bandwidth 1628 1793 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 209406 spinvalue up 0 responder packets 2 2 bytes 1628 1793 bandwidth 1628 1793 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 209411 spinvalue up 0 responder packets 2 3 bytes 1628 3073 bandwidth 1628 3073 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 209412 spinvalue up 0 responder packets 2 4 bytes 1628 4353 bandwidth 1628 4353 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 209447 spinvalue up 0 responder packets 2 5 bytes 1628 5633 bandwidth 1628 5633 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 220091 spinvalue up 1 initiator packets 2 6 bytes 1628 6913 bandwidth 1628 6913 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 360272 spinvalue up 0 responder packets 3 6 bytes 1759 6913 bandwidth 1759 6913 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 360522 spinvalue up 0 responder packets 3 7 bytes 1759 8193 bandwidth 1759 8193 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 360525 spinvalue up 0 responder packets 3 8 bytes 1759 9473 bandwidth 1759 9473 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 360527 spinvalue up 0 responder packets 3 9 bytes 1759 10753 bandwidth 1759 10753 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 360812 spinvalue up 0 responder packets 3 10 bytes 1759 12033 bandwidth 1759 12033 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 407997 spinvalue up 0 initiator packets 3 11 bytes 1759 13313 bandwidth 1759 13313 note "V.qn32,0-RTT,no spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 469185 spinvalue up 0 responder packets 4 11 bytes 1812 13313 bandwidth 1812 13313 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 469520 spinvalue up 0 responder packets 4 12 bytes 1812 13558 bandwidth 1812 13558 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 508049 spinvalue up 1 initiator packets 4 13 bytes 1812 14838 bandwidth 1812 14838 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 508049 measurement up full (initiator) 100052 packets 4 13 bytes 1812 14838 bandwidth 1812 14838 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 651750 spinvalue up 0 responder packets 5 13 bytes 1871 14838 bandwidth 1871 14838 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 652459 spinvalue up 0 responder packets 5 14 bytes 1871 16190 bandwidth 1871 16190 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 652463 spinvalue up 0 responder packets 5 15 bytes 1871 17470 bandwidth 1871 17470 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 652465 spinvalue up 0 responder packets 5 16 bytes 1871 18750 bandwidth 1871 18750 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 652466 spinvalue up 0 responder packets 5 17 bytes 1871 20030 bandwidth 1871 20030 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 656061 spinvalue up 1 initiator packets 5 18 bytes 1871 21310 bandwidth 1871 21310 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 784887 spinvalue up 1 responder packets 6 18 bytes 1930 21310 bandwidth 1930 21310 note "V.qn32,0-RTT,no R-spin"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 784887 measurement up right 276838 packets 6 18 bytes 1930 21310 bandwidth 1930 21310 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 784894 spinvalue up 1 responder packets 6 19 bytes 1930 22626 bandwidth 1930 22626 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 788969 spinvalue up 0 initiator packets 6 20 bytes 1930 23906 bandwidth 1930 23906 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 788969 measurement up left 4082 packets 6 20 bytes 1930 23906 bandwidth 1930 23906 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 788969 measurement up full (initiator) 280920 packets 6 20 bytes 1930 23906 bandwidth 1930 23906 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 969456 spinvalue up 1 responder packets 7 20 bytes 1988 23906 bandwidth 1988 23906 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 970079 spinvalue up 1 responder packets 7 21 bytes 1988 25186 bandwidth 1988 25186 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 970083 spinvalue up 1 responder packets 7 22 bytes 1988 26466 bandwidth 1988 26466 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 970084 spinvalue up 1 responder packets 7 23 bytes 1988 27746 bandwidth 1988 27746 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 970085 spinvalue up 1 responder packets 7 24 bytes 1988 29026 bandwidth 1988 29026 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 971603 spinvalue up 0 initiator packets 7 25 bytes 1988 30306 bandwidth 1988 30306 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1083636 spinvalue up 0 responder packets 8 25 bytes 2047 30306 bandwidth 2047 30306 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1083636 measurement up right 294667 packets 8 25 bytes 2047 30306 bandwidth 2047 30306 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1083636 measurement up full (responder) 298749 packets 8 25 bytes 2047 30306 bandwidth 2047 30306 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1083823 spinvalue up 0 responder packets 8 26 bytes 2047 31586 bandwidth 2047 31586 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1083830 spinvalue up 0 responder packets 8 27 bytes 2047 32866 bandwidth 2047 32866 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1275803 spinvalue up 0 responder packets 8 28 bytes 2047 34146 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1276003 spinvalue up 0 responder packets 8 29 bytes 2047 35426 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1276007 spinvalue up 0 responder packets 8 30 bytes 2047 36706 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1276008 spinvalue up 0 responder packets 8 31 bytes 2047 37986 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1276009 spinvalue up 0 responder packets 8 32 bytes 2047 39266 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1279399 packet up initiator length 59 packets 9 33 bytes 2106 40546 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1404524 spinvalue up 1 responder packets 9 33 bytes 2106 40546 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1404524 measurement up full (responder) 320888 packets 9 33 bytes 2106 40546 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1404533 spinvalue up 1 responder packets 9 34 bytes 2106 41826 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1404589 spinvalue up 1 responder packets 9 35 bytes 2106 43106 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1404592 spinvalue up 1 responder packets 9 36 bytes 2106 44386 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1561258 spinvalue up 1 responder packets 9 37 bytes 2106 45666 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1561264 spinvalue up 1 responder packets 9 38 bytes 2106 46946 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1561430 spinvalue up 1 responder packets 9 39 bytes 2106 48226 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1561434 spinvalue up 1 responder packets 9 40 bytes 2106 49506 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1561436 spinvalue up 1 responder packets 9 41 bytes 2106 50786 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1564548 spinvalue up 0 initiator packets 9 42 bytes 2106 52066 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1566372 packet up initiator length 58 packets 11 42 bytes 2222 52066 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1717952 spinvalue up 0 responder packets 11 42 bytes 2222 52066 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1717952 measurement up full (responder) 313428 packets 11 42 bytes 2222 52066 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1717960 spinvalue up 0 responder packets 11 43 bytes 2222 53346 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1718100 spinvalue up 0 responder packets 11 44 bytes 2222 54626 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1718102 spinvalue up 0 responder packets 11 45 bytes 2222 55906 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1718103 spinvalue up 0 responder packets 11 46 bytes 2222 57186 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1865639 spinvalue up 0 responder packets 11 47 bytes 2222 58466 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1865647 spinvalue up 0 responder packets 11 48 bytes 2222 59746 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1865650 spinvalue up 0 responder packets 11 49 bytes 2222 60190 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1873266 spinvalue up 1 initiator packets 11 50 bytes 2222 61470 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1873266 measurement up left 789630 packets 11 50 bytes 2222 61470 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1873266 measurement up full (initiator) 1084297 packets 11 50 bytes 2222 61470 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1933387 spinvalue up 1 responder packets 12 50 bytes 2275 61470 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1933387 measurement up right 60121 packets 12 50 bytes 2275 61470 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 1933387 measurement up full (responder) 215435 packets 12 50 bytes 2275 61470 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
QUIC 10.30.0.167 <-> 91.190.195.94 9b645a28-04b49f0f (56324:4433) at 2009530 spinvalue up 1 responder packets 12 51 bytes 2275 61529 bandwidth 2047 34146 note "V.qn32,0-RTT,spinning"
